# 02_basic256_not_BASIC256.diff by Ryan Kavanagh <ryanakca@kubuntu.org>
# Description: Let's call the executable basic256 instead of BASIC256 to keep
# consistent with previous releases.
Index: basic256/BASIC256.pro
===================================================================
--- basic256.orig/BASIC256.pro	2010-08-13 06:10:47.239664832 -0400
+++ basic256/BASIC256.pro	2010-08-13 06:10:55.583662694 -0400
@@ -3,7 +3,7 @@
 ######################################################################
 
 TEMPLATE		=	app
-TARGET			=	BASIC256
+TARGET			=	basic256
 DEPENDPATH		+=	.
 INCLUDEPATH		+=	.
 TMAKE_CXXFLAGS = -g 
